### PR TITLE
match cells across multiple days

### PR DESCRIPTION
**Pull request by inanhkn**
@kimtonyhyun 

`match_cell.m` matches the cells of any number of days provided as input. 

Example usage: (align cells across days 1,2,3 assuming `run_alignment `was run for 1,2 and 1,3)

`sources = {'path to day 1','path to day 2','path to day 3'}`
`m_base_to_others = {m1to2,m1to3}`
`matched_cells = match_cells(sources,m_base_to_others)`

### CODE REVIEW

File: alignment/match_cells.m [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+% Compute mutual matching matrix using pairwise cell overlap
+% matrices
+%
+% inputs:
+%   sources: 1 x [number of days] cell array containing paths to the
+%       folders of the days of interest. The base day (the one to which all
+%       other days are matched to) is taken to be the the first folder.
+%   m_base_to_others: 1 x [number of days-1] cell array where each entry has
+%       the pairwise overlap matrix between the base day and another.
+%       m_base_to_others{k} must match with sources{k+1}.
```
Comment (kimtonyhyun) [inline alignment/match_cells.m]: Just to be clear, this is the "overlap matrix" as returned by [`run_alignment`](https://github.com/inanhkn/analysis/blob/master/alignment/run_alignment.m#L81) as `info.overlap_matrix` (and _not_ `m1to2`)?

**Comment by inanhkn** [comment]
Actually, it is m1to2. I guess I didn't understand what you meant by the overlap matrix earlier.
I'll replace that with something more clear.

**Comment by kimtonyhyun** [comment]
How about turning `m_base_to_others` into a two-dimensional cell, like `m_itoj`? I am also exploring multiday matching now, and this seems to be a more natural representation than `m_base_to_others`.

File: alignment/match_cells.m [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+%
+% Hakan Inan (Mar 15)
+
+num_days = length(sources);
+classes = cell(1,num_days);
+
+% Get class labels
+for idx_day = 1:num_days
+    class_day = load_classification(get_most_recent_file(sources{idx_day}, 'class_*.txt'));
+    classes{idx_day} = strcmp(class_day,'not a cell') == 0;
```
Comment (kimtonyhyun) [inline alignment/match_cells.m]: `~strcmp(class_day, 'not a cell')` will also work

**Pull request closed without merging**